membership-attribute-service/app/controllers/AccountController.scala

```diff
@@ -271,6 +273,40 @@ class AccountController(commonActions: CommonActions, override val controllerCom
     }
   }
 
+
+  def allSubscriptions(contactRepo: SimpleContactRepository, subService: SubscriptionService[Future])(maybeUserId: Option[String]): OptionT[OptionEither.FutureEither, List[Subscription[SubscriptionPlan.AnyPlan]]] = for {
```

Comment:
could be worth splitting it down something along these lines
```suggestion
   def allSubscriptions(
     contactRepo: SimpleContactRepository,
     subService: SubscriptionService[Future]
   )(
     maybeUserId: Option[String]
   ): OptionT[OptionEither.FutureEither, List[Subscription[SubscriptionPlan.AnyPlan]]] =
   for {
```